From a29977ba0ca1e5617a24bc8693d033a085563ff3 Mon Sep 17 00:00:00 2001
From: Jules Aguillon <jules@j3s.fr>
Date: Tue, 12 Dec 2023 15:53:06 +0100
Subject: [PATCH] Don't make parser tests installable

This makes the regular build of odoc-parser to depend on its
test-dependencies, making it fail.
---
 src/parser/test/dune | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/parser/test/dune b/src/parser/test/dune
index 28c4d82f7..c6769550d 100644
--- a/src/parser/test/dune
+++ b/src/parser/test/dune
@@ -1,6 +1,5 @@
 (library
  (name odoc_parser_test)
- (package odoc-parser)
  (inline_tests)
  (enabled_if
   (>= %{ocaml_version} 4.04.1))
-- 
2.40.1

